packages/web-components/src/index.html

```diff
@@ -12,12 +12,12 @@
     <style type="text/css" media="screen">
       body {
           /* For a little breathing room around the components */
-          padding: 4rem;
+          /* padding: 4rem; */
       }
     </style>
 
   </head>
   <body>
-    <my-component first="Stencil" last="'Don't call me a framework' JS"></my-component>
-  </body>
+<va-minimal-footer />  
```

Comment:
I'm not sure I understand why this file was changed?

Reply:
ug, accidentally committed!